normandy/control/static/control/js/components/ActionForm.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { reduxForm } from 'redux-form'
+import { _ } from 'underscore'
+import HeartbeatForm from './action_forms/HeartbeatForm.jsx'
+import ConsoleLogForm from './action_forms/ConsoleLogForm.jsx'
+
+export class ActionForm extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    const { fields, arguments_schema, name } = this.props;
+
+    let childForm = 'No action form available';
+
+    switch(name) {
+      case 'show-heartbeat':
+        childForm = (<HeartbeatForm fields={fields} />);
+        break;
+      case 'console-log':
+        childForm = (<ConsoleLogForm fields={fields} />);
+        break;
+      default:
+        childForm = childForm;
+    }
+
+    return (
+      <div id="action-configuration">
+        <i className="fa fa-caret-up fa-lg"></i>
```

Comment:
Probably not for this PR but an `<FAIcon>` or `<Icon>` or `<FontAwesome>` component might be neat someday.